encoding/msgpack/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/philippgille/gokv/encoding/msgpack
```

Comment:
I liked the simplicity of using `encoding.JSON` or `encoding.Gob` from the convenience vars in https://github.com/philippgille/gokv/blob/4bd3d816ae29d840a95cfb38cf77c05b8485bae6/encoding/codec.go#L11. But that was ok as long as the codecs were just from the stdlib. I see it makes sense to separate the codecs into their own modules as soon as they have dependencies, to not pull in unnecessary dependencies of codecs that you don't use at all when using a `gokv.Store` with only a single codec.
Hm.